### PR TITLE
A few proposals / changes / fixes

### PR DESCRIPTION
**Pull request by Aurelien30000**
- Removed unused import
- Changed to parseInt in spawn subcommand (should be better)
- A renamed entity shouldn't be stacked again -> stack data deletion
- Only show the tag of a renamed entity on hover (as vanilla's behaviour)
- 2 new traits for Bees, I think they are interesting as long as they prevent quick production of honey or a massive bee attack ^^

Don't hesitate to comment ^^
I hope you'll enjoy it :)

### CODE REVIEW

**Comment by Aurelien30000** [comment]
Changelog updated !

File: src/main/java/uk/antiperson/stackmob/listeners/TagInteractListener.java [hunk 1]
```diff
@@ -35,5 +35,7 @@ public void onTagInteract(PlayerInteractEntityEvent event) {
             return;
         }
         stackEntity.slice();
+        stackEntity.removeStackData();
+        stackEntity.getEntity().setCustomNameVisible(false);
```
Comment (Nathat23) [inline src/main/java/uk/antiperson/stackmob/listeners/TagInteractListener.java]: Perhaps this should be moved to the #removeStackData method?

Comment (Aurelien30000) [reply src/main/java/uk/antiperson/stackmob/listeners/TagInteractListener.java]: Good idea, done ^^

**Comment by Aurelien30000** [comment]
- Update to Paper 1.15.2 for a future PR
- Improve TabCompletion to suggest according to the arg's beginning

**Comment by Aurelien30000** [comment]
Another reminder, sorry but you should merge this, important bugs are fixed :)

**Comment by andrewkm** [comment]
@Nathat23 @Aurelien30000 

> A renamed entity shouldn't be stacked again -> stack data deletion

This absolutely needs a config option.
We specifically **WANT** to stack renamed entities.

**Pull request merged**